README.md

```diff
@@ -5,3 +5,7 @@
 
 ## vector-embedding.py
     demonstrates Vector embedding phase of LLM pipeline ie phaseII
+
+
```

Comment:
Similarly for the following section, ensure that all section names start with ##.